### ISSUE TITLE
[TC-85] Should use "dest_ip" instead of "dest_domain" in "parent.config" and "cache.config" when ofqdn is ip

### ISSUE THREAD

**Issue by limited** [apache/incubator-trafficcontrol#890]
<p>Should use "dest_ip" instead of "dest_domain" in "parent.config" and "cache.config" when ofqdn is ip.</p>

Author: Jifeng Yang
JIRA Link: <a href="https://issues.apache.org/jira/browse/TC-85">https://issues.apache.org/jira/browse/TC-85</a>
Found Version: 2.1.0

**Comment by dneuman64** [comment]
I don't believe this will be included in 2.1, moving to 2.2.

**Comment by mitchell852** [comment]
@rob05c - do you think this is still relevant w/ atstccfg? how big of a deal is it if we use dest_domain instead of dest_ip? do you know?

**Comment by rob05c** [comment]
Yeah, it's still relevant. I don't think it's a huge issue, because we always use dest_domain, and MSO will put IPs in there, and it seems to work.

It's still something we should fix, because ATS says it's wrong, so it could theoretically break in the future.

We'll probably change TC config gen to always use Strategies, once we have them in ATS 9/10.

But until then, I'd vote keep this issue open, just to not forget about it.

**Comment by rob05c** [comment]
Copied from #7021 :

> it seems to work.

It doesn't work, that statement is wrong. I think we just didn't realize it before, because it falls back to the default, and we didn't used to have complex custom parents until recently.

This issue didn't capture the complete issue with the bug. This issue said we should do it, it doesn't say why. The reason and bug is that dest_domain=ip is silently ignored by ATS as if the entire line doesn't exist.
